.vnurc

```diff
@@ -12,3 +12,9 @@ Attribute “aria-posinset” not allowed on element “tr” at this point.
 Attribute “aria-setsize” not allowed on element “tr” at this point.
 # Ignoring role meter
 Bad value “meter” for attribute “role” on element “div”.
+
+# Warnings
```

Comment:
Can you make an issue to make sure they are "undisabled" in the future, and leave a link to that issue on this line? :)

Reply:
Created separate issues and added to file